packages/flutter/lib/src/rendering/auto_layout.dart

```diff
@@ -11,29 +11,38 @@ import 'object.dart';
 /// for constraints. Also sets up and manages implicit constraints and edit
 /// variables.
 class AutoLayoutRect {
```

Comment:
not necessarily in this PR, but the various members in this class could do with sample code showing how to use them.